### ISSUE TITLE
Parse identifiers with "bis" suffix

### ISSUE THREAD

**Issue by mico** [metanorma/pubid-itu#20]
I found some identifiers with "bis" suffix:
- `ITU-T X.50bis` (https://www.itu.int/rec/T-REC-X.50bis-198811-I/en)
- `ITU-T V.8bis` (https://www.itu.int/rec/T-REC-V.8bis/en)
- `ITU-T V.31bis` (https://www.itu.int/rec/T-REC-V.31bis-198811-I)

There are more, this is just few of them. The identifiers I looked into mentioned that it's "Extract from the Blue Book", maybe "bis" related to "Blue Book".

**Comment by opoudjis** [comment]
_bis_ is Latin for "twice", and it is used in numbering like A is: in general 8 bis means "8A, number I've had to slot in between 8 and 9". You can keep going with Latin times adverbs— _ter_, _quater_, _quinquies_ ... but we won't :)

**Comment by mico** [comment]
> _bis_ is Latin for "twice", and it is used in numbering like A is: in general 8 bis means "8A, number I've had to slot in between 8 and 9". You can keep going with Latin times adverbs— _ter_, _quater_, _quinquies_ ... but we won't :)

@ronaldtse should we convert "bis" to "A"?

**Comment by ronaldtse** [comment]
@mico we have to keep the “bis” as “bis”. Whatever the suffix is, we have to keep the original.

**Comment by opoudjis** [comment]
Yeah, I was only mentioning 8A as a more familiar way of doing what 8 bis does.
